src/shared_modules/rsync/tests/interface/CMakeLists.txt

```diff
@@ -9,12 +9,13 @@ link_directories(${CMAKE_BINARY_DIR}/lib)
 
 
 file(GLOB INTERFACE_UNITTEST_SRC
-    "*.cpp")
+    "*.cpp"
+    "${CMAKE_SOURCE_DIR}/src/*.cpp")
 
-add_executable(fim_integration_test 
+add_executable(rsync_unit_test 
     "${INTERFACE_UNITTEST_SRC}" )
 
-target_link_libraries(fim_integration_test 
+target_link_libraries(rsync_unit_test 
```

Comment:
add the check CMAKE_SYSTEM_NAME STREQUAL "Windows" and add the win target link_libraries